src/components/Portal/Portal.js

```diff
@@ -175,15 +175,23 @@ class Portal extends Component {
       rootNode,
       stopPropagation,
       stopPropagationEvents,
+      ...focusTrapOptions
     } = this.props;
 
+    const { fallbackFocus } = focusTrapOptions;
+
     return (
       isClient() &&
       createPortal(
         <FocusTrap
           active={focusTrap}
           focusTrapOptions={{
+            allowOutsideClick: () => true,
+            fallbackFocus:
+              fallbackFocus ||
+              (isClient() ? rootNode : Children.toArray(children)[0].type),
```

Comment:
Given that this is a major change in functionality, would it make sense to allow consumers to provide these options?

Reply:
I was torn about this - it seemed sensible to allow the outside click by default with the option to override it with `focusTrapOptions` so there would be no additional configuration to consumers, but perhaps that could have unintended side effects.

For the `fallbackFocus` I'm less sure what the right approach is here - with the package update that includes the configuration to fix this issue, the `focus-trap-react` package now expects a fallback or else it throws errors. What would be your recommendation here?

Reply:
I would say that `allowOutsideClick` changes would be considered breaking changes.

For the `fallbackFocus`, I think that your approach is good considering that it is a required option.

Reply:
Good point! In that case, we can pass through the `focusTrapOptions` prop from `PanelV2` - would that satisfy the requirement for this issue?